mix.exs

```diff
@@ -73,6 +73,7 @@ defmodule DotCom.Mixfile do
       {:address_us, "0.4.3"},
       {:aws, "1.0.4"},
       {:aws_credentials, "0.3.2", optional: true},
+      {:browser, "~> 0.5.4"},
```

Comment:
h/t to @thecristen for pointing out the existence of this library in [an earlier banner PR ](https://github.com/mbta/dotcom/pull/2175#pullrequestreview-2334036329) that I found while looking for prior art here!